homeassistant/components/derivative/sensor.py

```diff
@@ -63,7 +63,7 @@
 UNIT_PREFIXES = {
     None: 1,
     "n": 1e-9,
-    "µ": 1e-6,
+    "μ": 1e-6,  # "μ" == "\u03bc"
```

Comment:
Why do we need the comment # "μ" == "\u03bc", doesn't the pylint checker catch it? Same comment for other cases.

Reply:
Removed the comment